core/service/src/engine/utils.rs

```diff
@@ -22,6 +22,7 @@ where
 {
     // Query FHE key IDs
     let fhe_key_ids_set = match kms_type {
+        // TODO needs fixing to consider epoch ID
```

Comment:
Should it be in this PR or a follow up?  In the latter case we should add an issue

Reply:
should be fixed here https://github.com/zama-ai/kms/pull/336/commits/6d6ffc29371a3082b27e429643b2beb6e9007f1e